front-end/src/app/shared/pipes/fec-date.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DateTime } from 'luxon';
+
+@Pipe({
+  name: 'fecDate',
+})
+export class FecDatePipe implements PipeTransform {
+  transform(value: string): string {
+    const date = DateTime.fromFormat(value, 'yyyyMMdd');
```

Comment:
not something we'd want to deal with now but i would love to store dates as dates and worry about the fec `yyyymmdd` format only when we submit to fec.  storing data in mismatched types (e.g. dates in strings) when we don't really have to opens us up to a lot of pain

Reply:
This could be done in the model fromJSON() method, converting the yyyymmdd format into a Date type.